canvas_modules/common-canvas/src/common-properties/controls/expression/expression.scss

```diff
@@ -45,6 +45,16 @@
 			}
 		}
 
+		.cm-activeLine {
+			background-color: $background-selected; // works for both dark n light
+		}
+		.cm-activeLineGutter {
+			background-color: $background-active;
```

Comment:
This color doesn't look correct. Line Number is almost invisible in dark mode
<img width="215" alt="Screenshot 2024-05-15 at 1 10 56 PM" src="https://github.com/elyra-ai/canvas/assets/25124000/b1422f3d-98b8-45af-9a9e-50ec39c01c41">

Reply:
@nmgokhale i have given the color for dark mode in line #117-119. But it seems like the props will get updated only during the `light` toggle. Is there any way to change bg color based on theme? Because in this case, the same color token is not working good with both themes.
<img width="734" alt="image" src="https://github.com/elyra-ai/canvas/assets/151116685/7c60a59c-c133-4f74-97b7-f55733ed38b9">